### PR TITLE
Hue: install libmysqlclient20 for 2.0

### PR DESCRIPTION
**Pull request by functicons**

### CODE REVIEW

File: hue/hue.sh [hunk 1]
```diff
@@ -53,8 +54,20 @@ function update_repo() {
   if command -v apt-get >/dev/null; then
     retry_command "apt-get update"
   else
-    retry_command "yum -y update"
```
Comment (yigress) [inline hue/hue.sh:56]: if '-y' causing issue, can we add comment why not use it.

Comment (functicons) [reply hue/hue.sh]: Good catch. I should not have removed the `-y`.

**Review by yigress — APPROVED** [review]

**Pull request merged**